Eclipse_builtin.xml

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="21">
```

Comment:
Ist das jetzt das neue Profil oder das Eclipse 2.1?

Reply:
Dieses Version="21" war im export des Ecplise-Built-in. Ich glaube, dass ist die Version Profiles-XML Formats, nicht des inhalts. Ist einfach verwirrlich.